src/components/ItemMovie/ItemMovie.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Content } from '../../constants/common.constants';
+import { IMovie } from '../../types/interfaces';
+import classes from './ItemMovie.module.scss';
+
+interface IItemMoviesProps {
+  key: number;
+  movie: IMovie;
+  text: string;
+}
+
+class ItemMovie extends React.Component<IItemMoviesProps> {
+  public render(): JSX.Element | undefined {
+    const { name, poster, year, countries, rating } = this.props.movie;
+    const text = this.props.text;
+
+    if (
+      !name.includes(text) &&
+      !year.toString().includes(text) &&
+      !rating.toString().includes(text) &&
+      countries.filter((country) => country.name.includes(text)).length === 0
```

Comment:
there are too many checks, better to create a separate funtion or method that will check all this conditions

Reply:
put in a separate method